### PR TITLE
Cleanup deprecations

### PR DESCRIPTION
**Pull request by core23**
## Subject

Remove deprecated code and fix `NEXT_MAJOR` comments.

I am targeting this branch, because this is major cleanup.

### CODE REVIEW

File: src/Seo/SeoAwareTrait.php [hunk 1]
```diff
@@ -18,7 +18,7 @@ trait SeoAwareTrait
     /**
      * @required
      */
-    protected ?SeoPageInterface $seoPage;
+    protected ?SeoPageInterface $seoPage = null;
```
Comment (VincentLanglet) [inline src/Seo/SeoAwareTrait.php:21]: Why is this needed ? Null is the default value isnt it ?

Comment (core23) [reply src/Seo/SeoAwareTrait.php:21]: No: https://3v4l.org/DUJAP

**Review by VincentLanglet — APPROVED** [review]

**Review by wbloszyk — APPROVED** [review]
